sdk/core/core-http/review/core-http.api.md

```diff
@@ -545,10 +545,10 @@ export interface PolymorphicDiscriminator {
 }
 
 // @public @deprecated
-export function promiseToCallback(promise: Promise<any>): Function;
+export function promiseToCallback(promise: Promise<any>): (cb: Function) => void;
 
 // @public
-export function promiseToServiceCallback<T>(promise: Promise<HttpOperationResponse>): Function;
+export function promiseToServiceCallback<T>(promise: Promise<HttpOperationResponse>): (cb: ServiceCallback<T>) => void;
```

Comment:
Is this necessary?

Reply:
`Function` is rejected  by "@typescript-eslint/ban-types"